app/controllers/projects_controller.rb

```diff
@@ -87,4 +88,15 @@ def set_project
   def project_params
     params.require(:project).permit(:project_name)
   end
+
+  def responded(id)
+    count = 0
+    Participant.where(project_id: id).pluck(:last_responded).each do |answer|
```

Comment:
Nice way to get the number of people who responded here! Maybe clarify that this method returns a fraction of the people.